spec/bambora/www_form_client_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+module Bambora
+  describe WWWFormClient do
+    let(:api_key) { 'fakekey' }
+    let(:merchant_id) { 1 }
+    let(:base_url) { 'https://sandbox-api.na.bambora.com' }
+    let(:headers) do
+      {
+        'Content-Type' => 'application/x-www-form-urlencoded',
+        'Authorization' => 'Passcode MTpmYWtla2V5',
+      }
+    end
+    let(:path) { '/' }
+    let(:response_body) { { response: 'body' } }
+    let(:query_string_response_body) { URI.encode_www_form(response_body) }
+    let(:failed_status) { 500 }
+    let(:failed_response_body) { 'Stupid Garthim! You want Gelfling? Why not ask me?!' }
+
+    subject { described_class.new(base_url: base_url, merchant_id: merchant_id) }
+
+    describe '#post' do
+      let(:request_body) { { gelfling: 'Deet' } }
+
+      context 'server responds with a 2xx status' do
+        before do
+          stub_request(:post, base_url)
+            .with(headers: headers, query: request_body)
+            .to_return(headers: { 'Content-Type' => 'text/html' }, body: query_string_response_body)
+        end
+
+        it 'parses the response' do
+          resp = subject.post(path: path, body: request_body, api_key: api_key)
+          expect(resp).to eq response_body
+        end
+      end
+
+      # context 'server responds with a non 2xx status' do
```

Comment:
Why is this code commented out? Is it something that could be useful and we don't want to delete?

Reply:
Ah, oops! I had commented this out while debugging. Fixed!